PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -114,7 +114,7 @@ public void onCreate(Bundle savedInstanceState) {
 					public void onItemClick(AdapterView<?> arg0, View view,
 											int position, long id) {
 						if (answers.get(position).getNextQuestionID() > 0) {
-							// Next Question
+							// Next Questioni
```

Comment:
I assume this is a mistake. Please remove it for not to have typos in the comments